src/ansys/fluent/core/solver/flobject.py

```diff
@@ -169,6 +169,15 @@ def to_python_name(fluent_name: str) -> str:
     return name
 
 
+def _get_root_from_child_object(obj):
+    root = obj._parent
+    if root is None:
+        root = obj
+    else:
+        root = _get_root_from_child_object(root)
+    return root
+
+
 class Base:
     """Base class for settings and command objects.
```

Comment:
```suggestion
    """Provides the base class for settings and command objects.
```
Class descriptions should start with a verb ending in "s" or "es"